### ISSUE TITLE
Increased urinary neutrophil gelatinase-associated lipocalin

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4818]
**Preferred term label:**

Increased urinary neutrophil gelatinase-associated lipocalin
**Synonyms**
Increased urinary NGAL

**Definition (free text, please give PubMed ID)**
An increased concentration of  neutrophil gelatinase-associated lipocalin in the urine (there is no generally accepted threshold, but some studies choose a threshold of above 150 nanogram per milliliter).

Comment: An increased urinary NGAL concentration may indicate tubular injury.

PMID:18725016 

**Parent term (use hpo.jax.org/app)**

rearrange hierarchy -- Abnormal urine protein level
--proteinuria
-- specific proteins such as NGAL

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Better parent term
Abnormal urinary protein biomarker
def--Abnromal level of a specific protein in the urine -- todo work on this definition
